src/getData.js

```diff
@@ -12,16 +12,25 @@ module.exports.getProjectIDs = function getProjectIDs(requestedGroupID, token) {
     .getProjectsByGroupID(apiToken, groupID, getProjectSearchParams)
 
     .then((projects) => {
-      const ids = projects.map((project) => project.id);
-
-      return ids;
+      const projectList = [];
+      projects.map((project) => {
+        Project = {
+          id: project.id,
+          name: project.name,
+        };
+        projectList.push(Project);
```

Comment:
Bad code style. Missing "const". Variable name should be camelcase starts with small letters.

Reply:
I just found this style also appears somewhere else, pls help fix them altogether.